install/create_config_file.py

```diff
@@ -10,7 +10,7 @@
 bin_process_path = os.path.join(global_root, "workflow/scripts")
 qcore_path =  os.path.join(global_root, "qcore")
 tools_path = os.path.join(global_root, "tools")
-
+runfolder_path = "/home/melody.zhu/project/nesi00213/RunFolder" 
```

Comment:
hardcoded user specific path in a config?

Reply:
all changed back to global